plugins/modules/xml.py

```diff
@@ -327,37 +322,37 @@
     path: bar.xml
     xpath: /config/element[@name='test1']
     state: absent
-'''
+"""
 
-RETURN = r'''
+RETURN = r"""
 actions:
-    description: A dictionary with the original xpath, namespaces and state.
-    type: dict
-    returned: success
-    sample: {xpath: xpath, namespaces: [namespace1, namespace2], state=present}
+  description: A dictionary with the original xpath, namespaces and state.
+  type: dict
+  returned: success
+  sample: {xpath: xpath, namespaces: [namespace1, namespace2], state=present: null}
```

Comment:
I think this should be
```suggestion
  sample: {xpath: xpath, namespaces: [namespace1, namespace2], state: present}
```